docker-compose.yaml

```diff
@@ -82,10 +82,20 @@ services:
     networks:
     - ckan-multi
 
+  datapusher:
+    container_name: datapusher
+    build:
+      context: datapusher/
+    ports:
+      - "8800:8800"
+    networks:
+    - ckan-multi
+
   ckan:
     depends_on:
     - redis
     - solr
+    - datapusher
```

Comment:
`ckan` should not depend on `datapusher` globally, i.e. for all instances. The `depends_on` is an array that cannot be overwritten in override files, so if you put it here we're stuck with it for all themes.

If you define it as dependency in the override files it's enough, the override files append to this array but cannot replace it. (also valid for all docker-compose array types)

Reply:
Hope you're sure about that. I don't exactly remember why I included here